Update addText to accept arbitrary SSML wrapping elements

jnthnwn opened the pull request:
## Proposed changes
Changes to SpeechBuilder's addText method to support arbitrary SSML elements. #431 

These changes simplify adding other helper methods for SSML syntax and allow users to add SSML supported by a specific platform even if Jovo hasn't been updated its API with helper methods.

I'd like feedback on if it would make sense for another argument to added to the other public methods as well, so a user could addSentence with some surrounding SSML tags.

## Types of changes
<!--- Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply -->
<!--- You can also fill these out after creating the PR, or ask us, if you run into any problems! -->
- [X] My code follows the code style of this project
- [X] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [X] I have added tests to cover my changes
- [X] All new and existing tests passed

jankoenig commented:
Thank you!
We will take a deeper look early next week and provide feedback :)

Have a great weekend,
Jan

aswetlow commented:
Great PR! Thanks
Any idea of how to solve "self-closing" tags like `break` or `audio`?

I also like your idea about the `addSentence` method. Feel free to add more functions :)

jnthnwn commented:
added [this line](https://github.com/jovotech/jovo-framework/pull/442/files#diff-fdae62756d94c2f63bac9578691b8e45R205) to address self-closing tags.

i have added the surroundSsml argument to some of the convenience methods, but i think that adding more convenience methods might be a hassle to maintain later on. i think the changes to addText should be sufficient syntax for users to modify the speech.
